src/view/screens/Settings/index.tsx

```diff
@@ -730,7 +730,9 @@ export function SettingsScreen({}: Props) {
           onPress={() => openModal({name: 'change-password'})}
           accessibilityRole="button"
           accessibilityLabel={_(msg`Change password`)}
-          accessibilityHint={_(msg`Change your Bluesky password`)}>
+          accessibilityHint={_(
+            msg`Opens modal for change your Bluesky password`,
```

Comment:
I think "for changing" would work better here?
```suggestion
            msg`Opens modal for changing your Bluesky password`,
```